util.go

```diff
@@ -37,3 +40,44 @@ func prettyPrint(data interface{}) {
 	dbg, _ := json.MarshalIndent(data, "", "  ")
 	fmt.Println(string(dbg))
 }
+
+// GetDefaultRelease attempts to guess a default release.
+func GetDefaultRelease() string {
+	if release := os.Getenv("SENTRY_RELEASE"); release == "" {
```

Comment:
I think we can reduce the amount of indented code here by reversing the condition:

```go
if release := os.Getenv("SENTRY_RELEASE"); release != "" {
	return release
}

// continue trying to figure out a release name...
```

Reply:
I took the approach from your next note and discarded this block entirely